js/Paddle.js

```diff
@@ -4,30 +4,43 @@ class Paddle {
         this.ctx = canvasConfig.ctx; // pobranie wartości ctx z gettera
         this.paddleHeight = 10; // wysokość paletki
         this.paddleWidth = 75; //szerokość paletki
-        this.paddleX = (this.canvas.width-this.paddleWidth)/2; // pozycja paletki na osi X
+        this.paddleX = (canvasConfig.cvs.width-this.paddleWidth)/2; // pozycja paletki na osi X
+        this.canvasWidth = canvasConfig.cvs.width; // szerokość canvas
```

Comment:
nie potrzebnie odwołujesz się do do canvasConfig.cvs, masz je zdefiniowane wyżej w konstruktorze jako this.cvs, więc zmień na this.cvs.width